models/docs/sources/history_operations.md

```diff
@@ -1483,3 +1483,23 @@ The identifier of the second reserve asset in a liquidity pool or similar constr
 {% docs details_ledgers_to_expire %}
 The number of ledgers after which the operation will expire if its not executed.
 {% enddocs %}
+
+{% docs details_balance_id_strkey %}
+Strkey of the balance id
+{% enddocs %}
+
+{% docs details_claimable_balance_id_strkey %}
```

Comment:
Could we just reduce `details_balance_id_strkey` and `details_claimable_balance_id_strkey` to one doc key? Or will that cause problems if we have multiple fields reference the same doc key?